user_docs/en/changes.md

```diff
@@ -3,20 +3,20 @@
 ## 2024.3
 
 This release adds support for Unicode Normalization to speech and braille output.
-This can be useful when reading characters that are unknown to a particular speech synthesizer or braille table and which have a compatible alternative, like the bold and italic characters commonly uses on social media.
+This can be useful when reading characters that are unknown to a particular speech synthesizer or braille table and which have a compatible alternative, like the bold and italic characters commonly used on social media.

```

Comment:
This is strictly spoken unrelated. I think this is more suitable for a general documentation follow up pr. That said, I'm sorry for the typos.